src/models/StyledComponent.js

```diff
@@ -275,9 +278,29 @@ export default function createStyledComponent(target: Target, options: Object, r
    * forwardRef creates a new interim component, which we'll take advantage of
    * instead of extending ParentComponent to create _another_ interim class
    */
-  const WrappedStyledComponent = React.forwardRef((props, ref) => (
-    <ParentComponent {...props} forwardedComponent={WrappedStyledComponent} forwardedRef={ref} />
-  ));
+  const WrappedStyledComponent = React.forwardRef((props = EMPTY_OBJECT, ref) => {
+    // $FlowFixMe
+    const defaultProps = WrappedStyledComponent.defaultProps || EMPTY_OBJECT;
+    // $FlowFixMe
+    WrappedStyledComponent.defaultProps = {
```

Comment:
I could just merge `theme` here because it's only used here https://github.com/styled-components/styled-components/blob/d45ef9d0edb70a5b3bcbf74273f7acd5e8efdf9a/src/models/StyledComponent.js#L125
But since it's named `defaultProps`, i'm sending a proper defaultProps to `forwardedComponent`.